css-layout-classes/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>css layout classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-third">
+        <img class="characters" src="./images/link.png" alt="link" />
+      </div>
+      <div class="column-third">
+        <h2 class="column-third">Super Smash Bros. Ultimate</h2>
+        <p class="column-third">
```

Comment:
These shouldn't have a `column-third` class. They are inside of your `column-third`, they aren't a new column